src/lexer.coffee

```diff
@@ -845,31 +849,17 @@ exports.Lexer = class Lexer
     unless str[...closingDelimiter.length] is closingDelimiter
       @error "missing #{closingDelimiter}", length: delimiter.length
 
-    [firstToken, ..., lastToken] = tokens
```

Comment:
This was adjusting the location data of the first and last tokens of the interpolation contents to include the surrounding delimiters (eg the interpolated string `"` quotes)

That's not actually what we want AST-location-data wise

This change is what necessitates the updates to token location data tests in `tests/location.coffee`